backend(Payment)/App.js

```diff
@@ -14,25 +14,25 @@ function loadScript(src) {
             resolve(false);
         }
     })
-}
-const __DEV__ = 
+)}
+
+
+const __DEV__ = document.domain === 'localhost'
```

Comment:
users can chceck the localhost. when the localhost users run this applications, this does not charge you the real money.